### DISCUSSION TITLE
Power Automate OAuth Custom connector for ServiceNow redirect URL

### DISCUSSION THREAD

**Discussion by dewaltk** [microsoft/PowerPlatformConnectors#3252] [Q&A]
I am trying to use the sample custom connector ServiceNow OAuth with PowerAutomate posted by @vasavib, the information provides a redirect URL to use when creating the ServiceNow API but when I try to test the connector I get an invalid URL error message.

The redirect URL used is https://global.consent.azure-apim.net/redirect/

**Comment by troystaylor** [comment]
The ServiceNow connector is 3 years old and no longer maintained: 
https://github.com/microsoft/PowerPlatformConnectors/tree/dev/custom-connectors/ServiceNow
You could attempt to create your own custom connector from the Swagger file, but when you configure the OAuth you would need to select a unique Redirect URL - this is a recent change that Microsoft has implemented: 
https://learn.microsoft.com/en-us/connectors/custom-connectors/#21-oauth-20

**Comment by ozippy** [comment]
I found that once I generated the connector, it created the Unique redirect URL. I copied that and then updated the redirect URL in the ServiceNow App Registration and it then worked.
